src/utils/utils.py

```diff
@@ -80,3 +82,18 @@ def setup_logger():
         ],
     )
     return LOGGER
+
+
+def log_to_discord(scraper_outcomes: list[tuple[str, str]]):
```

Comment:
Could this be generalised? so that we can use this in other places, such as to send log messages in the middle of it.

Generally, I would prefer it to minimally have
- log levels
- a more generalised data structure such as list [str] that we can use for more generic logging purposes